scripts/linters/pre_commit_linter.py

```diff
@@ -558,14 +599,13 @@ def main(args=None):
         third_party_typings_linter.check_third_party_libs_type_defs(
             verbose_mode_enabled))
 
-    _print_complete_summary_of_lint_messages(lint_messages)
```

Comment:
I think we should output the success report as well so that users are aware of all the test the changes passed.

Reply:
The user always has the option to scroll up.

Reply:
Can you share the screenshot of the success case on develop and on this branch?

Reply:
**Before we had this**
![Screenshot from 2020-06-11 18-06-38](https://user-images.githubusercontent.com/29057268/84386091-5c859780-ac0e-11ea-8761-8fe8030dd3d8.png)
**Now we have this**
![Screenshot from 2020-06-11 18-08-13](https://user-images.githubusercontent.com/29057268/84386207-8d65cc80-ac0e-11ea-9e22-8b8f62b0518e.png)

The summary is: Now we do not have 'Summary of Errors' section if all the tests are passing.